client/src/main/java/io/pravega/client/segment/impl/SegmentOutputStreamImpl.java

```diff
@@ -80,6 +81,9 @@
     private final RetryWithBackoff retrySchedule;
     private final Object writeOrderLock = new Object();
     private final String delegationToken;
+    @VisibleForTesting
+    @Getter
+    private final long requestId = RequestIdGenerator.getRequestId();
```

Comment:
This does not seam useful here, as the CommandEncoder is going to transform many of these into an append block.

Reply:
Yes, the Command Encoder will use the `requestorId` which is part of the `Append`  to construct the `AppendBlock`.